lib/slack-ruby-bot/rspec/support/slack-ruby-bot/respond_with_error.rb

```diff
@@ -2,8 +2,6 @@
 
 RSpec::Matchers.define :respond_with_error do |error, error_message|
   match do |actual|
-    allow(Giphy).to receive(:random)
```

Comment:
This is going to break downstream users upgrading the library. I think this needs to be put back and have a `if defined?(Giphy)` or one of those.